packages/fast-tooling-react/src/form/custom-controls/control.file-upload.style.ts

```diff
@@ -0,0 +1,15 @@
+import { ComponentStyles, CSSRules } from "@microsoft/fast-jss-manager-react";
+
+export interface FileUploadControlClassNameContract {
+    fileUploadControl?: string;
+    fileUploadControl__disabled?: string;
+    fileUploadControl_input?: string;
+}
+
+const styles: ComponentStyles<FileUploadControlClassNameContract, {}> = {
```

Comment:
Since this is empty, do you really need it?

Reply:
We will, none of these are currently being exported and a design pass needs to be taken on them before we actually provide access to users.